Source/Aggregates/Builders/UnregisteredAggregateRoots.cs

```diff
@@ -39,7 +39,7 @@ void AddToContainer(TenantId tenantId, IServiceCollection serviceCollection)
     {
         foreach (var type in Types)
         {
-            serviceCollection.AddSingleton(
+            serviceCollection.AddTransient(
```

Comment:
This seems like a notable fix for the changelog?

Reply:
Yes, I was not quite finished yet 😁